doc/api/esm.md

```diff
@@ -946,9 +971,11 @@ customizations of Node.js’ code loading and evaluation behaviors.
 
 #### HTTPS loader
```

Comment:
This loader and the CoffeeScript one are in https://github.com/nodejs/loaders-test. Do those loaders work under this branch (maybe with `shortCircuit: true` added)?

Reply:
The https loader does not due to #42721. I'll confirm the CoffeeScript one still does.

Reply:
good news: nodejs/loaders-test#6